.github/workflows/ci_test-tpu.yml

```diff
@@ -16,81 +29,79 @@ env:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Get Current PR Number
+      id: vars
+      shell: bash
+      run: |
+        echo ::set-output name=PR_NUMBER::$(git ls-remote origin "pull/*/head" | grep -F -f <(git rev-parse HEAD) | awk -F'/' '{print $3}')
+
     - name: Install Go
       uses: actions/setup-go@v2
       with:
         go-version: 1.14.x
+
     - name: Set up Python 3.7
       uses: actions/setup-python@v2
       with:
         python-version: 3.7
 
-    - name: Checkout Pytorch Lightning
-      uses: actions/checkout@v2
-      with:
-        repository: PyTorchLightning/pytorch-lightning
-        ref: ${{ github.event.pull_request.head.sha }}
-
     - name: Checkout ml-testing-accelerators
       uses: actions/checkout@v2
       with:
         repository: GoogleCloudPlatform/ml-testing-accelerators
         path: ml-testing-accelerators
         ref: 5e88ac24f631c27045e62f0e8d5dfcf34e425e25
 
-    - name: Setup gcloud CLI
-      uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
-      with:
-        version: '290.0.1'
-        service_account_key: ${{ secrets.GKE_SA_KEY_BASE64 }}
-        project_id: ${{ secrets.GKE_PROJECT }}
-        export_default_credentials: true
-
-    # Configure Docker to use the gcloud command-line tool as a credential helper for authentication.
-    - name: Configure Docker
-      run: |-
-        gcloud --quiet auth configure-docker
-      shell: bash
-    - name: Build and Push Docker Image
-      env:
-        PYTHON_VER: ${{ matrix.python-version }}
-        XLA_VER: ${{ matrix.xla-version }}
-      run: |
-        #cd dockers/tpu-tests
-        docker build --tag "$IMAGE:$IMAGETAG" -f ./dockers/tpu-tests/Dockerfile --build-arg "PYTHON_VERSION=$PYTHON_VER" --build-arg "PYTORCH_VERSION=$XLA_VER" .
-        docker push "$IMAGE:$IMAGETAG"
-      shell: bash
-
     - name: Install jsonnet
       run: |-
         go get github.com/google/go-jsonnet/cmd/jsonnet
       shell: bash
+
     # Get the GKE credentials so we can deploy to the cluster
     # Use either zone or region depending on cluster setup.
-    - run: |-
+    - name: Get Cluster Credentials
+      run: |-
         gcloud container clusters get-credentials "$GKE_CLUSTER" --zone "$GKE_ZONE"
       shell: bash
 
     - name: Deploy the job on the kubernetes cluster
       env:
         XLA_VER: ${{ matrix.xla-version }}
       run: |-
-        python -c "fname = 'dockers/tpu-tests/tpu_test_cases.jsonnet' ; ttt = open(fname).read().replace('pytorch-VERSION', 'pytorch-$XLA_VER') ; open(fname, 'w').write(ttt)"
-        job_name=$(jsonnet -J ml-testing-accelerators/ dockers/tpu-tests/tpu_test_cases.jsonnet --ext-str image=$IMAGE --ext-str image-tag=$IMAGETAG | kubectl create -f -) && \
+        python -c "fname = 'dockers/tpu-tests/tpu_test_cases.jsonnet' ; ttt = open(fname).read().replace('{PYTORCH_VERSION}', '$XLA_VER').replace('{PR_NUMBER}', '${{ steps.vars.outputs.PR_NUMBER }}') ; open(fname, 'w').write(ttt)"
```

Comment:
here I'd recommend passing PR number using `--ext-str pr_number=...` 

you might be able to use `--ext-str pr_number=$(steps.vars.outputs.PR_NUMBER)` or maybe you'd need to stick `steps.vars.outputs.PR_NUMBER` into an ENV var so you could use `--ext-str pr_number=PR_NUMBER`